Gems/ROS2/Code/Source/VehicleDynamics/ModelLimits/SkidSteeringModelLimits.cpp

```diff
@@ -40,6 +42,20 @@ namespace ROS2::VehicleDynamics
                         "Angular speed Limit",
                         "Max angular speed (rad/s)")
                     ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::Default,
+                        &SkidSteeringModelLimits::m_angularAccleration,
+                        "Angular acceleration",
+                        "Acceleration in rad/s²")
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::Default,
+                        &SkidSteeringModelLimits::m_linearAcceleration,
+                        "Linear acceleration",
+                        "Acceleration in m/s²")
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
                     ->Attribute(AZ::Edit::Attributes::Max, 10.0f);
```

Comment:
consider limits since we do not want users to not be able to simulate things that the engine can